ffi/capi/c/examples/segmenter_tiny/Makefile

```diff
@@ -78,4 +78,4 @@ test: build
 
 clean:
 	git clean -xf *
-	rm -f ../../../../../target/debug/libicu_capi_staticlib.a
+	rm -f crate/target/debug/libcrate.a
```

Comment:
Observation: `make clean` is now inconsistent between fixeddecimal_tiny and segmenter_tiny